packages/zcli-themes/src/commands/themes/preview.ts

```diff
@@ -96,13 +93,16 @@ export default class Preview extends Command {
     ]
 
     const watcher = chokidar.watch(monitoredPaths).on('change', async (path) => {
-      console.log(chalk.bold.gray('Change'), path)
-      if (await preview(themePath, flags)) {
+      this.log(chalk.bold('Change'), path)
+      try {
+        await preview(themePath, flags)
         wss.clients.forEach((client) => {
           if (client.readyState === WebSocket.OPEN) {
             client.send('reload')
           }
         })
+      } catch (e: any) {
+        this.error(e, { exit: false })
```

Comment:
Good idea catching errors here, I see that you've added a test for this as well 💯